examples/image_streamer/golden_image.pp

```diff
@@ -66,7 +67,8 @@
   require => Image_streamer_golden_image['golden_image_5'],
   data    => {
     name                       => 'Golden_Image_2',
-    golden_image_download_path => 'golden_image.zip',  # can be either an absolute or relative path
+    golden_image_download_path => 'golden_image_downloaded.zip',  # can be either an absolute or relative path
+    force                      => true
```

Comment:
Should add a comment explaining what force does. Also should be included in resources.md